src/ResourceManager/FrontDoor/Commands.FrontDoor/Cmdlets/RemoveAzureRmFrontDoorContent.cs

```diff
@@ -0,0 +1,64 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Collections;
+using System.Management.Automation;
+using System.Net;
+using Microsoft.Azure.Commands.FrontDoor.Common;
+using Microsoft.Azure.Commands.FrontDoor.Helpers;
+using Microsoft.Azure.Commands.FrontDoor.Models;
+using Microsoft.Azure.Commands.FrontDoor.Properties;
+using Microsoft.WindowsAzure.Commands.Utilities.Common;
+using Microsoft.Azure.Management.FrontDoor;
+using System.Linq;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+namespace Microsoft.Azure.Commands.FrontDoor.Cmdlets
+{
+    /// <summary>
+    /// Defines the New-AzureRmFrontDoor cmdlet.
+    /// </summary>
+    [Cmdlet("Remove", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "FrontDoorContent", SupportsShouldProcess = true), OutputType(typeof(void))]
```

Comment:
Please follow the same convention for the output as this: https://github.com/hytao/azure-powershell/blob/frontdoor/src/ResourceManager/FrontDoor/Commands.FrontDoor/Cmdlets/RemoveAzureRmFrontDoor.cs

Reply:
Do I need to add -PassThru for this?

Reply:
Yes

Reply:
Added